### ISSUE TITLE
Problem with right prompt and line height

### ISSUE THREAD

**Issue by developej** [jeremyFreeAgent/oh-my-zsh-powerline-theme#6]
Hey,

I already contacted jeremy via email (didn't know about the issues tab, I'm new in this git stuff), but since I didn't figure it out (and since someone else might also have this issue) I'm posting it here:

I have problem with right prompt - it looks like this:
http://i.imgur.com/4a3pl.png
there's a space between arrow and date/time background, but also, the arrow height is not right (arrow height is the problem on the left side too). In email, jeremy told something about line height not set right, but I didn't figure it out - I googled for urxvt line height and zsh line height - no results...

tl;dr - I want arrows to be the same height as the background colors and I don't want spaces between arrows and background colors.
I am using urxvt with Inconsolata for Powerline, size 11.

Thanks!
